concordium-consensus/src/Concordium/GlobalState/Basic/BlockState/Accounts.hs

```diff
@@ -223,8 +223,8 @@ deserializeAccounts migration cryptoParams = do
                   | cred `Map.member` regids = fail "Duplicate credential"
                   | otherwise = return $ Map.insert cred acctId regids
             newRegIds <- foldM addRegId accountRegIds $
-                (ID.credId <$> Map.elems (acct ^. accountCredentials))
-                ++ removedCredentialsToList (acct ^. accountRemovedCredentials . unhashed)
+                (ID.toRawCredRegId . ID.credId <$> Map.elems (acct ^. accountCredentials))
+                ++ (ID.toRawCredRegId <$> removedCredentialsToList (acct ^. accountRemovedCredentials . unhashed))
```

Comment:
It appears to me that we don't have to do this conversion here. We could just have removed credentials as raw credentials.

But we can also merge it as is, since I am making related changes to the account structure in the followup PR. What do you prefer?

Reply:
Yes.